crowbar_framework/db/schema.rb

```diff
@@ -11,9 +11,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150312081459) do
+ActiveRecord::Schema.define(version: 20150806114015) do
 
-  create_table "proposals", force: :cascade do |t|
+  create_table "proposal_queues", force: true do |t|
+    t.string "barclamp"
+    t.string "name"
+    t.text   "properties"
```

Comment:
Style/ExtraSpacing: Unnecessary spacing detected.<br>Style/SingleSpaceBeforeFirstArg: Put one space between the method name and the first argument.